canonical/ubuntu.com
# Clean up /credentials

## Done

- Remove edX API
- Rename things from cube to cred
- Fix linting issues

## QA

Ensure that the following pages still load:
- https://ubuntu-com-12393.demos.haus/credentials
- https://ubuntu-com-12393.demos.haus/credentials/syllabus
- https://ubuntu-com-12393.demos.haus/credentials/self-study
- https://ubuntu-com-12393.demos.haus/credentials/sign-up
- https://ubuntu-com-12393.demos.haus/credentials/exit-survey
- https://ubuntu-com-12393.demos.haus/credentials/schedule
- https://ubuntu-com-12393.demos.haus/credentials/your-exams
- https://ubuntu-com-12393.demos.haus/credentials/provision

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
